sensor_msgs/PointCloud2 combined_cloud
---
gpd_ros/GraspConfigList grasp_msg_list
